internal/templates/index.html

```diff
@@ -0,0 +1,280 @@
+                    </template>
+                </select>
+
+                <select 
+                    @change="handleChangeFilter('isPaid', $event)"
+                    id="isPaid" 
+                    class="appearance-none flex-1 bg-white border border-gray-300 rounded-lg py-2 pl-4 pr-4 text-gray-700 cursor-pointer hover:border-gray-400 focus:outline-none focus:ring-2 focus:ring-blue-500"
+                >
+                    <option value="">Preço</option>
+                    <option :value="false">Gratuito</option>
+                    <option :value="true">Pago</option>
+                </select>
+            </div>
+
+        </section>
+
+        <section class="mt-4 flex flex-wrap gap-2 mb-4">
+            <template x-for="(values, key) in currentFilters">
+                <template x-for="value in Array.from(values)">
+                    <span class="inline-flex items-center gap-1 px-3 py-1 rounded-full text-sm bg-blue-100 text-blue-800">
+                        <span x-text="formatFilter(key, value)"></span>
+                        <button @click="changeFilter(key, value)" class="cursor-pointer ml-1 text-blue-600 hover:text-blue-800">
+                            <i class="fas fa-times"></i>
+                        </button>
+                    </span>
+                </template>
+            </template>
+        </section>
+
+        <main class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4">
+            <template
+                x-for="(item, index) in filteredItems"
+                :key="index"
+            >
+                <article class="bg-white rounded-lg shadow-sm p-6 flex flex-col hover:shadow-md transition-shadow duration-200 border border-gray-100">
+                    <div class="flex items-start justify-between">
+                        <h3 x-text="item.Title" class="text-lg font-semibold text-gray-900"></h3>
+                        <span class="bg-blue-50 text-blue-800 text-xs font-medium px-2.5 py-0.5 rounded-full" x-text="item.IsPaid ? 'Pago' : 'Gratuito'"></span>
+                    </div>
+    
+                    <p class="mt-1 text-sm text-gray-600 mb-2">Por <span x-text="item.Author"></span></p>
+                    
+                    <div class="flex flex-wrap gap-2 mt-auto mb-4">
+                        <template x-for="tag in item.Tags">
+                            <span class="bg-gray-100 text-gray-800 text-xs font-medium px-2.5 py-0.5 rounded-full" x-text="tag"></span>
+                        </template>
+                        <span class="bg-gray-100 text-gray-800 text-xs font-medium px-2.5 py-0.5 rounded-full" x-text="item.Type"></span>
+                        <span class="bg-gray-100 text-gray-800 text-xs font-medium px-2.5 py-0.5 rounded-full" x-text="item.Level"></span>
+                        <span class="bg-transparent text-gray-600 border border-gray-200 text-xs font-medium px-2.5 py-0.5 rounded-full" x-text="item.Language"></span>
```

Comment:
Não tava entendendo por que a tag da língua tinha outra cor. Testei com cinza e achei bom,  o que tu acha?

```suggestion
                        <span class="bg-gray-100 text-gray-800 text-xs font-medium px-2.5 py-0.5 rounded-full" x-text="item.Language"></span>
```

![image](https://github.com/user-attachments/assets/9442f90a-6177-4cdf-a10f-0843c6b18e9b)

Reply:
Boa, tinha feito de outra cor pra dar um destaque, mas com o emoji da bandeira já chama a atenção